mm2src/mm2_main/tests/docker_tests/docker_tests_inner.rs

```diff
@@ -5471,3 +5473,82 @@ fn test_approve_erc20() {
 
     block_on(mm.stop()).unwrap();
 }
+
+#[test]
+fn test_peer_time_sync_validation() {
+    let timeoffset_tolerable = TryInto::<i64>::try_into(MAX_TIME_GAP_FOR_CONNECTED_PEER).unwrap() - 1;
+    let timeoffset_too_big = TryInto::<i64>::try_into(MAX_TIME_GAP_FOR_CONNECTED_PEER).unwrap() + 1;
```

Comment:
nit: casting to `i64` with `TryInto`s and `unwrap`s seems unnecessary since we are only using this value to convert it into a `String`.

Reply:
I intended to have it as i64 to indicate it's possible to emulate time deviation in both directions, if anyone needs